WooCommerce/Resources/AppStoreStrings.pot

```diff
@@ -87,31 +87,31 @@ msgstr ""
 msgctxt "app_store_screenshot-2"
 msgid ""
 "Create orders\n"
-"on the fly\n"
+"on the fly"
 msgstr ""
 
 #. translators: This is a promo message that will be attached on top of a screenshot in the App Store.
 #. No specified characters limit here, but try to keep as short as the source one.
 msgctxt "app_store_screenshot-3"
 msgid ""
 "Take payments\n"
-"in person\n"
+"in person"
 msgstr ""
 
 #. translators: This is a promo message that will be attached on top of a screenshot in the App Store.
 #. No specified characters limit here, but try to keep as short as the source one.
 msgctxt "app_store_screenshot-4"
 msgid ""
 "Add and edit products\n"
-"with a touch\n"
+"with a touch"
 msgstr ""
 
 #. translators: This is a promo message that will be attached on top of a screenshot in the App Store.
 #. No specified characters limit here, but try to keep as short as the source one.
 msgctxt "app_store_screenshot-5"
 msgid ""
 "Get notified of\n"
-"every sale\n"
+"every sale"
```

Comment:
The various `\n` removals were unexpected. The linter didn't complain so I think they'd be effectively harmless. Still, here's a PR to fix them before merging on `trunk` to avoid noise in GlotPress.

https://github.com/woocommerce/woocommerce-ios/pull/10545